support-frontend/assets/components/marketingConsent/marketingConsentWithCheckbox.jsx

```diff
@@ -112,7 +113,25 @@ class MarketingConsentWithCheckbox extends Component<PropTypes, StateTypes> {
   }
 
   render() {
-    const emailFrequency = this.props.countryGroupId === 'AUDCountries' ? 'an occasional' : 'a weekly';
+    const createTitleText = () => {
+      const { countryGroupId, contributionType } = this.props;
+      const titleTexts = {
+        UsAllPlusRowRecurring: 'a weekly email',
+        AuAll: 'an occasional email',
+        UkRowSingle: 'occasional emails',
```

Comment:
Is there meaningful difference between "an occasional email" and "occasional emails" ? @ionamckendrick

Reply:
Iona confirmed 'occasional emails' can be used